### PR TITLE
Updated the error messages

### LINKED ISSUE #433: crane: error messages are unhelpful
```
$ crane pull ubuntu
Error: accepts 2 arg(s), received 1
Usage:
  crane pull [flags]
```

What it's complaining about is the lack of an arg to specify output path, but you wouldn't be able to tell from the error message, only from reading generated md docs.

A better error message would be something like `Error: must specify output path` with `Usage: crane pull [image] [output-path]`

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Fixes #433

### CODE REVIEW

File: pkg/crane/pull.go [hunk 1]
```diff
@@ -34,12 +34,25 @@ func init() { Root.AddCommand(NewCmdPull()) }
 
 // NewCmdPull creates a new cobra.Command for the pull subcommand.
 func NewCmdPull() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "pull",
```
Comment (jonjohnsonjr) [inline pkg/crane/pull.go]: I think you can just update `Use` to achieve the same thing.

Comment (hpandeycodeit) [reply pkg/crane/pull.go]: right. I don't why I did that. Reverted it now.

File: pkg/crane/pull.go [hunk 2]
```diff
@@ -35,10 +35,15 @@ func init() { Root.AddCommand(NewCmdPull()) }
 // NewCmdPull creates a new cobra.Command for the pull subcommand.
 func NewCmdPull() *cobra.Command {
 	return &cobra.Command{
-		Use:   "pull",
+		Use:   "crane pull [image] [output-path]",
 		Short: "Pull a remote image by reference and store its contents in a tarball",
-		Args:  cobra.ExactArgs(2),
-		Run:   pull,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 2 {
```
Comment (u5surf) [inline pkg/crane/pull.go]: We  also should handle when length of args is 0. So we can use switch statement instead of ifs.

Comment (hpandeycodeit) [reply pkg/crane/pull.go]: Is there a different error message when args equal to 0?

File: pkg/crane/pull.go [hunk 3]
```diff
@@ -35,10 +35,15 @@ func init() { Root.AddCommand(NewCmdPull()) }
...
+				return errors.New("must specify output path")
```
Comment (jonjohnsonjr) [inline pkg/crane/pull.go]: This doesn't compile (missing `errors` import).

**Comment by jonjohnsonjr** [comment]
Closing in favor of https://github.com/google/go-containerregistry/pull/584

**Pull request closed without merging**